src/current/v25.1/time.md

```diff
@@ -133,12 +133,12 @@ Comparing `TIME` values:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> CREATE TABLE time (time_id INT PRIMARY KEY, time_val TIME(4));
+> CREATE TABLE IF NOT EXISTS time_precise (time_id INT PRIMARY KEY, time_val TIME(4));
 ~~~
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> SHOW COLUMNS FROM time;
+> SHOW COLUMNS FROM time_precise;
 ~~~
 
 ~~~
```

Comment:
In case it's important to reflect, the output has changed slightly:

~~~
  column_name | data_type | is_nullable | column_default | generation_expression |       indices       | is_hidden
--------------+-----------+-------------+----------------+-----------------------+---------------------+------------
  time_id     | INT8      |      f      | NULL           |                       | {time_precise_pkey} |     f
  time_val    | TIME(4)   |      t      | NULL           |                       | {time_precise_pkey} |     f
~~~

Reply:
updated, thanks!